pkg/client/unversioned/discovery_client.go

```diff
@@ -154,7 +126,9 @@ func (d *DiscoveryClient) ServerResources() (map[string]*unversioned.APIResource
 
 func setDiscoveryDefaults(config *Config) error {
 	config.Prefix = ""
-	config.Version = ""
+	config.Version = "unversioned"
```

Comment:
I'll leave a todo and clean it up later. I believe we'll make more changes to RESTClient in the near future.